Assets/MixedRealityToolkit-Tests/InteractionDefinitionTests.cs

```diff
@@ -24,7 +24,7 @@ public void Test01_TestObjectChanged()
             var testValue1 = (object)1f;
             var testValue2 = (object)false;
 
-            var initialValue = inputDef.GetRaw();
+            var initialValue = inputDef.GetRawValue();
```

Comment:
These were here to show Simon the difference in performance. The tests are largely duplicated if you update these methods.

We can just remove altogether.